tests/UI/specs/MultiSites_spec.js

```diff
@@ -13,12 +13,38 @@ describe("MultiSitesTest", function () {
     var generalParams = 'idSite=1&period=year&date=2012-08-09';
     var selector = '.pageWrap,.expandDataTableFooterDrawer';
 
+    var createdSiteId = null;
+
+    before(function (done) {
+        var callback = function (undefined, response) {
```

Comment:
The first argument to the callback will be an error (ie, it's a node.js style callback). If defined, you should call `done(error)` (where error is `undefined` renamed).